app/assets/javascripts/dashboard/dashboard_task_list_view.js

```diff
@@ -45,6 +46,8 @@ type State = {
   showFinishedTasks: boolean,
   finishedTasks: Array<APITaskWithAnnotation>,
   unfinishedTasks: Array<APITaskWithAnnotation>,
+  finishedTasksPageNumber: int,
```

Comment:
I wonder why flow does not complain, but this should be number. JS doesn't really know ints.